cds/modules/deposit/api.py

```diff
@@ -106,7 +106,6 @@ def _dumps(obj):
                 'size': obj.file.size if obj.file else 0,
                 'file_id': str(obj.file_id),
                 'completed': obj.file is not None,
-                'progress': 100 if obj.file is not None else '',
```

Comment:
can you update the tests then? The only one failing is because it expects this key